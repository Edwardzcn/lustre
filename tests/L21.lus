(**)A(*tests*)


node f(i : int) returns (j : int)
let
  j = i;
tel

