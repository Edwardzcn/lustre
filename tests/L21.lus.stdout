L21.lus:4:1: Parse error.
